contracts/examples/digital-cash/src/pay_fee_and_fund.rs

```diff
@@ -0,0 +1,43 @@
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use crate::{constants::*, helpers, storage};
+
+#[multiversx_sc::module]
+pub trait PayFeeAndFund: storage::StorageModule + helpers::HelpersModule {
+    #[endpoint]
+    #[payable("*")]
+    fn pay_fee_and_fund(&self, address: ManagedAddress, valability: u64) {
```

Comment:
this does not work with eGLD. Add implementation for that.

Reply:
in case of eGLD you have only 1 payment. so according to tokenID - make different implementaiton.

Reply:
we still have the classic deposit_fees (call 1) -> fund (call 2) 
this was not changed. This new implementation of pay_fee_and_fund comes extra for ESDT fund and fee only